<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sarra</class>
 <widget class="QDialog" name="sarra">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>1171</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(58, 58, 58);
color: rgb(255, 255, 255);
selection-background-color: rgb(183, 134, 32);
selection-color: rgb(0, 0, 0);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Livreur</string>
    </attribute>
    <widget class="QTableView" name="tableView_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>441</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter  Livreur</string>
     </property>
     <widget class="QLabel" name="ID_LIVREUR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>id livreur</string>
      </property>
     </widget>
     <widget class="QLabel" name="NOM_LIVREUR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>nom livreur</string>
      </property>
     </widget>
     <widget class="QLabel" name="PRENOM_LIVREUR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>prenom livreur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditIDL">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditNomL">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditPrenomL">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QLabel" name="TEL_LIVREUR">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>TEL</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTelL">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonTriDescL">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Tri Desc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonValidL">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Ajouter Livreur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModifL">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>230</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Modifier Livreur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSuppL">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>370</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Supprimer Livreur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAscL">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>360</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Tri Asc </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRechL">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCamL">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStatL">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModifL">
     <property name="geometry">
      <rect>
       <x>940</x>
       <y>270</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxRechL">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>400</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditModiftelephone">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRechL">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>340</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TIME_Livreur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TimeLivreur&#10;">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSuppL">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>340</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tarif</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonTriAscTarif">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Tri Asc </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonModifTarif">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>200</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Modifier Tarif</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCamTarif">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonTriDescTarif">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Tri Desc</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>60</y>
       <width>441</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajouter  Tarif</string>
     </property>
     <widget class="QLabel" name="CODE_TARIF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>code tarif</string>
      </property>
     </widget>
     <widget class="QLabel" name="DATE_TARIF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>date tarif</string>
      </property>
     </widget>
     <widget class="QLabel" name="SOMME_TARIF">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: pink;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>Somme tarif</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCT">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonValidTarif">
      <property name="geometry">
       <rect>
        <x>32</x>
        <y>170</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
      <property name="text">
       <string>Ajouter Tarif</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditSMAjout">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonStattarif">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>380</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Statistiques</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>401</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRechTarif">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ModifTarif">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>240</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRechTarif">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>300</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNvCodeT">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNvSommeT">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>time tarif</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TimeTarif">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxRechTarif">
     <property name="geometry">
      <rect>
       <x>920</x>
       <y>370</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSuppTarif">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>370</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: black;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSuppTarif">
     <property name="geometry">
      <rect>
       <x>472</x>
       <y>370</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid black;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
border-radius: 10px;
padding: 0 8px;
background: blue;
font: bold, &quot;Comic Sans MS&quot;;;
text-align:center;</string>
     </property>
     <property name="text">
      <string>Supprimer Tarif</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
